.github/workflows/release.yaml

```diff
@@ -255,7 +235,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: [ prepare ]
     env:
-      IMAGE: gcr.io/${{ secrets.GCR_REPOSITORY_DEPLOYER }}:${{ needs.prepare.outputs.version_with_major_minor }}
+      IMAGE: gcr.io/${{ secrets.GCR_REPOSITORY_DEPLOYER }}:${{ needs.prepare.outputs.version }}
```

Comment:
out of curiosity, why you switch back to full version?

Reply:
For context, we discussed previously and decided it would be nice to have a full version with no prefix (like line 239 and version_with_major_minor).